seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-druid/src/main/java/org/apache/seatunnel/druid/sink/DruidOutputFormat.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.seatunnel.flink.sink;
+package org.apache.seatunnel.druid.sink;
```

Comment:
Did you miss the flink package distinction？

Reply:
Yes, fixed this.